Framework/src/Framework/Process.cxx

```diff
@@ -329,6 +330,11 @@ void Process::run() {
         masterFile = &inFile;
       }
 
+      // In case we'd like to skip up to the event of minEvents_
+      while (n_events_processed < (minEvents_ - 1) && masterFile->nextEvent()) {
```

Comment:
Since we are skipping these events, we want to tell `nextEvent` to *not* store the event in the output file (if there is an output file).

```suggestion
      while (n_events_processed < (minEvents_ - 1) && masterFile->nextEvent(false)) {
```